examples/express-composition/src/observers/hello.observer.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/example-express-composition
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  lifeCycleObserver, // The decorator
+  LifeCycleObserver,
+} from '@loopback/core';
+
+/**
+ * This class will be bound to the application as a `LifeCycleObserver` during
+ * `boot`
+ */
+@lifeCycleObserver('')
```

Comment:
nitpick: maybe give it a name?

Reply:
Please note the 1st arg is NOT the name but the group for sorting.